### PR TITLE
[no-jira] Attempt switch from Travis to GitHub Actions

### PR DESCRIPTION
**Pull request by knutsenm**

### CODE REVIEW

**Comment by knutsenm** [comment]
I will adjust the branch production rules after code review.

**Review by jbirdjavi — APPROVED** [review]

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      - name: Check out files
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Bundle audit
+        run: bundle exec bundle audit check --update --ignore CVE-2015-9284
```
Comment (frett) [inline .github/workflows/ruby.yml:61]: Any reason we are ignoring a CVE here? I didn't see it ignored in the Travis config

**Review by frett — COMMENTED** [review]
Just the question about the CVE

File: .github/workflows/ruby.yml (hunk 1 above)
Comment (knutsenm) [reply .github/workflows/ruby.yml:61]: That is copied over from other Travis configs, for uniformity, but it only applies to OmniAuth and so doesn't matter here.

**Review by frett — APPROVED** [review]

**Comment by knutsenm** [comment]
Note that the bash uploader underlying the `codecov` gem is deprecated and going away early next year. We should replace it with v2 of https://github.com/marketplace/actions/codecov

**Pull request merged**